rust/module.rs

```diff
@@ -452,6 +452,11 @@ pub fn module(ts: TokenStream) -> TokenStream {
 
     format!(
         "
+            /// The module name.
+            ///
+            /// Used by the printing macros, e.g. [`info!`].
+            const __MODULE_NAME: &[u8] = b\"{name}\";
```

Comment:
Wouldn't it be better to guarantee that the string is null terminated here?

Reply:
We don't use the null -- do you mean just in case?

Reply:
In case we want to use it for something else. But if we do this, I think it would make sense to not pass the size to `printk`, just `%s` and the null terminated string...

Reply:
Agreed, will do.